src/olympia/reviewers/management/commands/review_reports_sql/addon/risk.sql

```diff
@@ -0,0 +1,30 @@
+SELECT risk_category AS `Risk Category`,
+       FORMAT(SUM(n), 0) AS `All Reviewers`,
+       FORMAT(SUM(CASE WHEN `gruppe` = 'volunteer' THEN n ELSE 0 END), 0) AS 'Volunteers'
+FROM
+  (SELECT CASE
+              WHEN weight > @RISK_HIGHEST THEN 'highest'
+              WHEN weight > @RISK_HIGH THEN 'high'
+              WHEN weight > @RISK_MEDIUM THEN 'medium'
+              ELSE 'low'
+          END AS risk_category,
+          gruppe,
+          COUNT(*) AS n
+   FROM editors_autoapprovalsummary aa
+   JOIN
+     (SELECT version_id,
+             CASE WHEN user_id NOT IN
+        (SELECT user_id
+         FROM groups_users
+         WHERE group_id IN
+             (SELECT id
+              FROM groups
+              WHERE name IN ('Staff', 'No Reviewer Incentives'))) THEN 'volunteer' ELSE 'all' END AS `gruppe`
```

Comment:
I think we should stick to English wording, "gruppe" -> "group"

Reply:
I am happy to use something else, but `group` is a reserved word, as is `inner` which is why I used `iner` as an alias for some inner queries.

Reply:
`group_category` maybe?

Reply:
Also, aliasing an inner query as `iner` (or "inner" which is what you want to do) doesn't really tell the story of what that inner query represents and what data it fetches. Aliases should be descriptive and tell the story of what data they hold imho.